services/fee-estimator/package.json

```diff
@@ -26,7 +26,8 @@
     "start": "node app.js",
     "clean": "rm -rf ./node_modules",
     "test": "npm run test:unit",
-    "test:unit": "jest --config=jest.config.js --detectOpenHandles --forceExit"
+    "test:unit": "jest --config=jest.config.js --detectOpenHandles --forceExit",
+    "test:functional": "jest --config=jest.config.js --detectOpenHandles --forceExit"
```

Comment:
Why are we using the same jest config for both unit and functional tests? This seems incorrect.